lib/top_hangman/round.rb

```diff
@@ -1,9 +1,23 @@
+require_relative 'render'
+
 module TopHangman
   class Round
     attr_reader :word
 
     def initialize(word)
       @word = word
+      @end_of_round = false
+      @guesses = []
+      @guess = nil
+      @guess_count = 0
```

Comment:
isn't this the same as `@guesses.size`?

Reply:
No, but I think it (`@guess_count`) isn't self-explanatory. Whit it; I want to count mistakes, so maybe `@mistakes_count`

Reply:
Ohh I see, `@mistakes_count` makes sense
How do you plan to identify a `guess` as a mistake or assertion?